homeassistant/components/unifiprotect/entity.py

```diff
@@ -46,7 +40,7 @@ def _async_device_entities(
     klass: type[ProtectDeviceEntity],
     model_type: ModelType,
     descs: Sequence[ProtectRequiredKeysMixin],
-    unadopted_descs: Sequence[ProtectRequiredKeysMixin],
+    unadopted_descs: Sequence[ProtectRequiredKeysMixin] | None = None,
     ufp_device: ProtectAdoptableDeviceModel | None = None,
 ) -> list[ProtectDeviceEntity]:
```

Comment:
This function could use more cleanup, but I didn't want to expand the scope anymore as its already a larger PR than I had hoped